LoopKitUI/Views/Settings Editors/PresentedAlert.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  PresentedAlert.swift
+//  LoopKitUI
+//
+//  Created by Rick Pasetto on 8/3/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+
+enum PresentedAlert {
```

Comment:
If we defer auth errors to the modal, we can return this to the ScheduleEditor namespace. If you want to extract it, I think we should rename it for clarity; something like SettingSaveAlert.

Reply:
I'll rename it... but I kind of like it as a reusable enum...